hwdetect/hwdetect.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+function echoDetected() {
+    
+    if [ -s $2 ]
+    then
+        echo $1:
+        cat $2
+    else
+        echo No $1 detected.
+    fi
+}
+
+function echoDiff {
+    
+    ls /dev | grep $2 > temp_diff.txt
+    local tmp_diff=$(diff temp_diff.txt $1)
+    
+    if [ ! -z "$tmp_diff" ]
+    then
+        echo $tmp_diff
```

Comment:
Nice. But there is garbage (from the task POV) characters at the beginning of line.
Add
`| cut -f 2- -d " "`
to print fields from 2 to end of string.

Reply:
Applied. Looks much better now:
```
< ttyUSB0
> ttyUSB0
< mmcblk0 < mmcblk0p1
> mmcblk0 > mmcblk0p1
```